crates/anvil/tests/it/traces.rs

```diff
@@ -73,8 +88,18 @@ contract Contract {
     let (abi, bytecode, _) = contract.into_contract_bytecode().into_parts();
 
     let (_api, handle) = spawn(NodeConfig::test()).await;
-    let provider = handle.ethers_ws_provider();
-    let wallets = handle.dev_wallets().collect::<Vec<_>>();
+    let provider = ethers_ws_provider(&handle.ws_endpoint());
+    let alloy_wallets = handle.dev_wallets().collect::<Vec<_>>();
+    let wallets = alloy_wallets
+        .into_iter()
+        .map(|w| {
+            Wallet::new_with_signer(
+                w.signer().clone(),
+                w.address().to_ethers(),
+                w.chain_id().unwrap(),
+            )
+        })
+        .collect::<Vec<_>>();
```

Comment:
this is duplicated a lot, not a fan of having this everywhere.

I think we could solve this with an extension trait for testing, same with the ethers_provider function